ss/util/modules.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+var Modules = []string{
+	"aclaccesscontrol",
+	"authz",
+	"acc",
+	"bank",
+	"capability",
+	"distribution",
```

Comment:
do we need to include dex for historical purposes? since the module used to exist, doing a backfill for the XOR hash should include that, right?